Resources/Prototypes/Entities/Objects/Tools/welders.yml

```diff
@@ -50,7 +50,7 @@
     qualities: Welding
   - type: Welder
     litMeleeDamageBonus:
-      types: # When lit, negate standard melee damage and replace with heat
+      types:
         Heat: 10
         Blunt: -10
```

Comment:
Doesn't this just heal 5 blunt damage on-hit now

Reply:
whoops